Engine/EditorUI/Panel/PanelConfiguration.cpp

```diff
@@ -291,6 +294,26 @@ void PanelConfiguration::ShowRenderOptions()
 			}
 			ImGui::TreePop();
 		}
+
+		ImGui::Separator();
+
+		ImGui::TextColored(ImVec4(1, 1, 0, 1), "Lighting");
+		ImGui::SameLine();
+		ImGui::TextColored(ImVec4(1, 1, 1, 1), "and");
+		ImGui::SameLine();
+		ImGui::TextColored(ImVec4(0.3, 0.3, 0.3, 1), "Shadows");
+
+		ImGui::Separator();
+
+		ImGui::TextColored(ImVec4(1, 1, 0, 1), "Ambient Light");
+
+		ImGui::SliderFloat("Intensity", &App->lights->ambient_light_intensity, 0, 1, "%.2f");
+		ImGui::ColorEdit3("Color", App->lights->ambient_light_color);
```

Comment:
This parameters of ambient light are not persistent, so we cannot change them and save the changes. To do that, we have to modify directly the class `ModuleLight`.

We should create a new light or find a way to store this values in  a file in order to modify them easily